.travis.yml

```
@@ -1,11 +1,28 @@
 sudo: false
-dist: xenial
 language: python
-python:
-  - "3.4"
-  - "3.5"
-  - "3.6"
-  - "3.7"
-install: pip install tox-travis
+install: 
+        - pip3 install -r requirements.txt -r requirements-extra.txt
 script: tox
-
+matrix:
+        include:
+                - os: "linux"
+                  dist: "xenial"
+                  python: "3.5"
+                - os: "linux"
+                  dist: "xenial"
+                  python: "3.6"
+                - os: "linux"
+                  dist: "xenial"
+                  python: "3.7"
+                - os: "windows"
+                  language: shell
+                  python: "3.7"
+                  env: "PATH=/c/python37:/c/python37/Scripts:$PATH"
+                  before_install:
+                          - choco install python make
+                          - cp /c/python37/python.exe /c/python37/python3.exe
+                  before_script:
+                          - pip install -r requirements-windows.txt
```

Unless there is a reason not to, I think using `pip3` for all the pip commands would be more consistent.
```suggestion
                          - pip3 install -r requirements-windows.txt
```

---

Done